### DISCUSSION TITLE
Bruno - Knowledge Hub 🧠

### DISCUSSION THREAD

**Discussion by helloanoop** [usebruno/bruno#386] [Show and tell]
Share Your Expertise with the Community!

Whether you're creating
- 📷 Video tutorials
- 📖 Case studies
- 📝 Blog posts

This is your space to contribute and inspire. Share your creations here and foster a thriving community of learning.

**Comment by cyrilgeorge153** [comment]
Blog on REST API Testing with Bruno
https://medium.com/@cyrilgeorge153/rest-api-testing-with-bruno-2656f92737e4
Video tutorials on Bruno in English.
https://www.youtube.com/playlist?list=PL9ucy3ZNv-8A4evd-c8EaFt-F3nHd8xFR

**Comment by DivyMohan14** [comment]
Blog on: Why I switched from Postman to Bruno 
https://divymohanrai.com/blog/ditch_postman

**Comment by naodeng** [comment]
Blog on: Introducing Bruno for Replacement Postman
[https://inaodeng.com/en/guild/api-testing/bruno/introduction/](https://inaodeng.com/en/guild/api-testing/bruno/introduction/)

Blog on: Bruno API Automation Testing Tutorial: Building a Bruno API Automation Test project from 0 to 1
[https://inaodeng.com/en/guild/api-testing/bruno/building-project/](https://inaodeng.com/en/guild/api-testing/bruno/building-project/)

Demo project: [https://github.com/Automation-Test-Starter/Bruno-API-Test-Starter](https://github.com/Automation-Test-Starter/Bruno-API-Test-Starter)

More API Testing Guild:[https://inaodeng.com/en/guild/#api-testing](https://inaodeng.com/en/guild/#api-testing)

**Comment by michelboaventura** [comment]
I’ve created a POC app that migrates from bru to yml. I’ve tested it only on my projects and it worked flawlessly, but it could fail with some not expected formats. https://github.com/michelboaventura/bru2oc/

For Mac releases I still need to sign it so they macOS doesn’t complain about unsigned apps.
